### ISSUE TITLE
Freeze

### ISSUE THREAD

**Issue by applibs** [dryabov/phpregexp#10]
After update of PHPStorm with this plugin, PHPStorm freeze when using CTRL + F and edits in popup window.

**Comment by dryabov** [comment]
In which PhpStorm release, stable or EAP?

**Comment by applibs** [comment]
stable 2019.1.3

**Comment by dryabov** [comment]
What file (project) are you working with? In my test it works smooth (quite a large project with one file containing over 80 regexps matches and replaces). Could you send me (dryabov@yandex.ru) the file that results in the freeze with a short description of manipulations you did in the search box to produce this freeze?

**Comment by applibs** [comment]
It will say nothing. I sent you phpstorm logs.

**Comment by iVegas** [comment]
I faced the same issue. Disabling the plugin resolve freezes.
Steps to reproduce:
- clone Drupal core repo
```
git clone --branch 8.8.x https://git.drupalcode.org/project/drupal.git
cd drupal
```
- open the file `core/includes/install.core.inc`;
- try to edit or "find in the file" some value;

**Comment by dryabov** [comment]
@iVegas Thank you! Fixed in 0.9.5.

**Comment by iVegas** [comment]
Updated an extension - freezes not appear anymore.
